atlite/gis.py

```diff
@@ -727,10 +727,27 @@ def compute_availabilitymatrix(
         total=len(shapes),
         desc="Compute availability matrix",
     )
+
     if nprocesses is None:
+        logger.info(
+            "You can disable the progress bar by setting 'disable_progressbar=True'."
+        )
+
+        if not disable_progressbar:
+            tqdm_kwargs = dict(
+                ascii=False,
+                unit=" gridcells",
+                total=len(shapes),
+                desc="Compute availability matrix",
+            )
+            iterator = tqdm(shapes.index, **tqdm_kwargs)
+        else:
+            logger.info("Progress bar is disabled.")
+            iterator = shapes.index
+
```

Comment:
```suggestion
        if not disable_progressbar:
            iterator = tqdm(shapes.index, **tqdm_kwargs)
        else:
            iterator = shapes.index
```
Great @Irieo , I think we can make it more minimalistic right?

Reply:
Yes!